package-res/ccc/core/cartesian/axis/abstract-cart-axis-panel.js

```diff
@@ -582,8 +582,16 @@ def
         // in case _calcContinuousBandSizeMin does it for us.
         layoutInfo.ticks = layoutInfo.ticksText = null;
 
+        var anchorLength = this.anchorLength();
+        var clientLengthMin = 0;
+
+        var plotSizeMin = !this.chart.parent ? this.chart.options.plotSizeMin : null;
```

Comment:
Passing the argument through `pvc_Size.to(plotSizeMin).resolve(null)` is necessary for the case when `plotSizeMin` is a number (`n` gets translated to `{width: n, height: n}`).